charts/datafold/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: datafold
 description: Helm chart package to deploy Datafold on kubernetes.
 type: application
-version: 0.1.2
```

Comment:
Do we automate this already?

Reply:
@cfernhout : It's manual. Bit boring, but the linter checks it for us. Not sure how to automate this correctly tbh. You mean a pre-commit?

Reply:
https://linear.app/datafold/issue/CX-2082/todo-at-end-of-milestone